src/wallet/rpcwallet.cpp

```diff
@@ -2458,13 +2458,13 @@ UniValue fundrawtransaction(const UniValue& params, bool fHelp)
                             "     \"changePosition\"    (numeric, optional, default random) The index of the change output\n"
                             "     \"includeWatching\"   (boolean, optional, default false) Also select inputs which are watch only\n"
                             "     \"lockUnspents\"      (boolean, optional, default false) Lock selected unspent outputs\n"
-                            "     \"feeRate\"           (numeric, optional, default 0=estimate) Set a specific feerate (fee per KB)\n"
+                            "     \"feeRate\"           (numeric, optional, default not set: makes wallet determine the fee) Set a specific feerate (satoshis per KB)\n"
```

Comment:
Not this pull's fault, but I wonder why was it chosen to use satoshis ere instead of just BTC/kB (converted using the AmountFromValue function)? This is inconsistent with the rest of the API.